WooCommerce/Classes/ViewRelated/Products/Media/ProductImagesCollectionViewController.swift

```diff
@@ -267,3 +282,12 @@ private extension ProductImagesCollectionViewController {
                                 forCellWithReuseIdentifier: InProgressProductImageCollectionViewCell.reuseIdentifier)
     }
 }
+
+/// Private data structures
+/// 
+private extension ProductImagesCollectionViewController {
+    enum Localization {
+        static let dragAndDropMessageWhileUploading = NSLocalizedString("Please wait until all uploads are finished before reordering images.",
+                   comment: "Alert message to inform the user that they must wait for all image uploads to complete before they can reorder the images.")
+    }
```

Comment:
Nit: Please follow the new convention to use reverse-DNS keys for localized strings (p91TBi-aJl-p2)

Reply:
Good catch! Fixed here: 54396e1c564a8b4edcbffd299d29e861f231206f

Reply:
Ah I always forgot this 💯

Reply:
Wonder if we can add this into code checking rule.

Reply:
Good idea! I think someone suggested that too but it will require updating all the localized strings in the app at once, which is not a trivial task for both us devs and translators 😅